src/test/java/me/karakelley/http/publicdirectory/PublicDirectoryTest.java

```diff
@@ -26,7 +26,7 @@ void testMissingDirectory() {
   @Test
   void testPathPointsToAFile() {
     TempFilesHelper.withTempDirectory(directory -> {
-      Path file = TempFilesHelper.createTempFile(directory, "/test1");
+      Path file = TempFilesHelper.createTempFile(directory, "/test1.txt");
```

Comment:
Because before I had hardcoded `.txt` in the `TempFilesHelper` but in order to create other type of files, i.e. html files which is necessary for this story, I needed to change the implementation to be more dynamic